src/components/uiOptions/js/UIOptions.js

```diff
@@ -26,4 +26,28 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
         ]
     });
 
+    /** A configuration of UIOptions which is suitable for statically localised contexts. It accepts two additional
+     * top-leve options,
+     * {String} locale - The initial locale in which UIOptions should render
+     * {String} direction - A suitable value for the `dir` attribute of the UIOptions container - this may take
+     * values `ltr`, `rtl` or `auto`
```

Comment:
Do you think we should default to `auto`?

Reply:
We default to not setting it, which I think is reasonable - I'll clarify in the comments